.gitignore

```diff
@@ -1,3 +1,4 @@
 *.o
 /arbor-gui
 *.ini
+*.kdev4
```

Comment:
That seems highly specific.

Reply:
KDevelop, not that specific.

Reply:
I would prefer not to list every IDE config file here. There's quite a lot of them ;)

Reply:
I'm just asking for KDev config ;)

Reply:
Well. This one time only.

Reply:
:bow: